### PR TITLE
Updates of language and a bit of structure

### PR DESCRIPTION
**Pull request by ctb**
cc @ljcohen @halexand

### CODE REVIEW

File: Alexander_etal_OpEd_Reanalaysis.tex [hunk 1]
```diff
@@ -52,11 +70,11 @@ \section{Background}
...
+In fact, a substantial proportion of these data products might be aptly categorized as `dark data,' as they are largely undiscoverable and often archived independently in association with a publication or on private servers. Even more limiting, however, is that the guidelines for public databases such as the TSA specifically state that ``Assemblies from sequences not directly sequenced by the submitter'' should not be uploaded to the TSA, thereby excluding the potential for reassembled datasets to be made available and directly linked to preexisting BioProjects, BioSamples, TSAs, and SRA entries.\footnote{@CTB do we have a citation/link for this policy?}
```
Comment (halexand) [inline Alexander_etal_OpEd_Reanalaysis.tex:75]: Yes, it is in the first paragraph on this page: https://www.ncbi.nlm.nih.gov/genbank/tsa/

I guess we can just link to the website? Seems somewhat static?

**Comment by ctb** [comment]
yes

File: Alexander_etal_OpEd_Reanalaysis.tex [hunk 2]
```diff
@@ -67,17 +85,73 @@ \subsection{Directly linking secondary data products to provenance of work-flow}
...
-
-Beyond the optimization of computational resources, the greatest opportunity for scientific advancement with high-throughput sequencing projects lies within our ability to make data products `forward discoverable.' In an ideal future, a researcher might be automatically notified when a dataset that she is actively working on is updated or changes. This presents many social and technical challenges that will need to be solved if we are to take full advantage of public data sets.
-
-% One can imagine several ways that this might happen, such as Zenodo or Open Science Framework (OSF) adding utility to provide users the option of following a DOI chain that will automatically notify the user if any alterations or updates occur (e.g. a new DOI is provided for a repository). Another, less user-dependent method might involve the use of JSON-LD files to automatically track the occurrence of the original raw data DOI in the wild.
+The Github-Zenodo framework presented here represents a relatively low
```
Comment (halexand) [inline Alexander_etal_OpEd_Reanalaysis.tex:113]: I like this reorganization-- it makes a more linear read and flows better.

File: Alexander_etal_OpEd_Reanalaysis.tex (hunk 1 above)
Comment (halexand) [reply Alexander_etal_OpEd_Reanalaysis.tex:75]: @ctb Thoughts?

**Comment by ctb** [comment]
> Yes, it is in the first paragraph on this page: https://www.ncbi.nlm.nih.gov/genbank/tsa/
> 
> I guess we can just link to the website? Seems somewhat static? 

yep.

File: Alexander_etal_OpEd_Reanalaysis.tex [hunk 3]
```diff
@@ -30,17 +30,35 @@
...
+Ultimately, such iterations on the original raw data have the
+potential to improve upon the secondary data products -- the assembled
+transcriptomes and associated annotations that are relied upon by the
+broader community for biological inquiry.
+
+Through this process, we developed several practices that we believe
+to be broadly applicable when reanalyzing data, especially when done
+by small research groups.
+
+% @CTB how publish
```
Comment (halexand) [inline Alexander_etal_OpEd_Reanalaysis.tex:61]: If asking how to publish on gigasci: 

> Submission through Overleaf
> For creating manuscripts in LaTeX, Gigascience recommends the use of its own LaTeX class files. Our class files are available online at Overleaf and also as a downloadable package via the links below.
> 
> Overleaf is a free, collaborative online LaTeX editor that allows you to write your manuscript in a TeX or rich text environment, to generate PDF outputs as you write, and to share your manuscript with co-authors and collaborators. When ready for submission, the manuscript files can be downloaded from Overleaf and submitted to the journal’s online submission system.

Comment (halexand) [reply Alexander_etal_OpEd_Reanalaysis.tex:61]: I will pull down the format quickly and get it going.

**Comment by halexand** [comment]
I think it is good to merge! @ljcohen?

**Comment by halexand** [comment]
Didn't mean to close it!

**Pull request merged**